app/lib/services.rb

```diff
@@ -30,4 +30,10 @@ def self.worldwide_api
   def self.registries
     Registries::BaseRegistries.new
   end
+
+  if ENV["FEATURE_FLAG_ACCOUNTS"] == "enabled"
+    def self.accounts_api
+      ENV.fetch("GOVUK_ACCOUNT_URI")
```

Comment:
What's the best naming convention here? endpoint? URI? URL?
Wasn't sure if this might end up in Plek and I should match what was there?

Reply:
I think go for `PLEK_SERVICE_ACCOUNT_MANAGER_URI`, as that's what `Plek.find("account-manager")` would look for

Reply:
much better idea, thanks.
amended [here](https://github.com/alphagov/finder-frontend/pull/2207/commits/0c49583d4d3091799987921bd7bc2797b5feba81#diff-ee054dafd7eadee4c8baf208077209baR36)